### PR TITLE
feature:create a get started screen

### PR DESCRIPTION
**Pull request by Lalkovich**

### CODE REVIEW

File: lib/get_started/get_started_view.dart [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class CustomBackgroundPainter extends CustomPainter {
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: move to separate file

File: lib/get_started/get_started_view.dart [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                      onPressed: () {},
+                      style: ElevatedButton.styleFrom(
+                        padding: const EdgeInsets.symmetric(vertical: 16.0,horizontal: 32.0),
+                        foregroundColor: const Color(0xFFf3e1d4),
+                        backgroundColor: const Color(0xFFf3e1d4),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(30),
+                        ),
+                      ),
+                      child: const Text('Get Started',style: TextStyle(color: Color(0xFFaa9184),fontSize: 18),),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: image
image

Font style is different

File: lib/get_started/get_started_view.dart [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      backgroundColor: const Color(0xFFf2e9e0),
+      body: CustomPaint(
+        size: const Size(double.infinity, double.infinity),
+        painter: CustomBackgroundPainter(),
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 8.0),
+          child: Stack(
+            children: [
+              Positioned(top: 0,right: -24,
+                  child: Image.asset('assets/images/tarelka-removebg-preview.png',height: 300,width: 300,fit: BoxFit.cover,)),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: image
No images import in commit and on even if they have to be they going to override the text below

Comment (Andestrial123) [reply lib/get_started/get_started_view.dart]: + i mentioned that images have padding from right side . They shouldn't have it

File: lib/get_started/get_started_view.dart [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                        foregroundColor: const Color(0xFFf3e1d4),
+                        backgroundColor: const Color(0xFFf3e1d4),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(30),
+                        ),
+                      ),
+                      child: Text(
+                        'Get Started',
+                        textScaler: TextScaler.linear(ScaleSize.textScaleFactor(context)),
+                        style: TextStyle(color: Colors.brown,fontSize:ScreenUtil().setSp(16)),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: image
Text steel not as a styled on design

File: lib/get_started/get_started_view.dart [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+                  SizedBox(height: screenHeight * 0.05),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart:141]: image
Image going throw system bar

**Review by Andestrial123 — DISMISSED** [review]

File: lib/get_started/get_started_view.dart [hunk 6]
```diff
@@ -87,10 +82,8 @@ class GetStartedView extends StatelessWidget {
                       alignment: Alignment.topLeft,
                       child: Padding(
                         padding: const EdgeInsets.only(left: 20.0),
-
-                        ///TODO move to localization file @Lalcovich
                         child: Text(
-                          'Lorem ipsum dolor sit amet,\nconsectetur adipiscing elit sed.',
+                          'lorem_1'.tr(),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: Move to const name of string key

File: lib/get_started/get_started_view.dart [hunk 7]
```diff
@@ -103,10 +96,8 @@ class GetStartedView extends StatelessWidget {
                       alignment: Alignment.topLeft,
                       child: Padding(
                         padding: const EdgeInsets.only(left: 20.0),
-
-                        ///TODO move to localization file @Lalcovich
                         child: Text(
-                          'Tempor incididunt ut labore et \ndolore magna aliqua consectetur \nadipiscing elit sed.',
+                          'lorem_2'.tr(),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: Move to const name of string key

File: lib/get_started/get_started_view.dart [hunk 8]
```diff
@@ -128,10 +119,8 @@ class GetStartedView extends StatelessWidget {
                             borderRadius: BorderRadius.circular(30),
                           ),
                         ),
-
-                        ///TODO move to localization file @Lalcovich
                         child: Text(
-                          'Get Started',
+                          'get_started'.tr(),
```
Comment (Andestrial123) [inline lib/get_started/get_started_view.dart]: Move to const name of string key

**Review by Andestrial123 — APPROVED** [review]

**Review by Andestrial123 — APPROVED** [review]

File: lib/get_started/get_started_view.dart (hunk 2 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: font style updated

File: lib/get_started/get_started_view.dart (hunk 3 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: pubspec was updated with assets

File: lib/get_started/get_started_view.dart (hunk 5 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart:141]: Andestrial fix this issue and made it flexible

File: lib/get_started/get_started_view.dart (hunk 6 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: moved to locale keys

File: lib/get_started/get_started_view.dart (hunk 7 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: moved to locale keys

File: lib/get_started/get_started_view.dart (hunk 8 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: moved to locale keys

File: lib/get_started/get_started_view.dart (hunk 4 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: Text style was updated and look like in design

File: lib/get_started/get_started_view.dart (hunk 1 above)
Comment (Lalkovich) [reply lib/get_started/get_started_view.dart]: Custom background moved to separate file

**Pull request merged**